.github/workflows/push-deploy.yml

```diff
@@ -1,5 +1,8 @@
 name: Deploy to WordPress.org
 
+permissions:
+  contents: read
```

Comment:
@dkotter should probably update this to `write` as well as add?
> packages: write

Reply:
Same comment as a few other repos, we're not actually attaching a release asset as part of this workflow so unless we want to add that, I don't think changes are needed here